perception/point_cloud_object_detection_and_recognition/tools/labeling_tool/label_model.cpp

```diff
@@ -0,0 +1,399 @@
+#include "label_model.hpp"
+#include "main_window.cpp"
```

Comment:
So the idea of this file is to label bags with PCODAR clusters so we know what they are in the future? This opens up some interesting possibilities for machine learning down the road. Were there any other plans for this? To use this to label clusters in real time?